mkdocs.yml

```diff
@@ -10,7 +10,7 @@ nav:
         - Magnetoencephalography: modality-specific-files/magnetoencephalography.md
         - Electroencephalography: modality-specific-files/electroencephalography.md
         - Intracranial Electroencephalography: modality-specific-files/intracranial-electroencephalography.md
-        - Task events: modality-specific-files/task-events.md
+        - Events: modality-specific-files/events.md
```

Comment:
Should also update the redirect on L86. Since the redirects haven't been in a released version, I don't think we need another redirect from `modality-specific-files/task-events.md` to `modality-specific-files/events.md`.

Reply:
yeap, just did that! pushed rewrite